sharedgo/deletionPicker.go

```diff
@@ -0,0 +1,21 @@
+// Picks what file from 10 to 50 (files 0-9 kept in the system for persistence) to delete
+
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"strconv"
+	"time"
+)
+
+func pick_file_to_delete() string {
```

Comment:
Use camelCase for function names.

```suggestion
func pickFileToDelete() string {
```